sdk/src/main/java/io/opentelemetry/sdk/metrics/DoubleObserverSdk.java

```diff
@@ -33,9 +42,32 @@
         monotonic);
   }
 
+  @Override
+  List<MetricData> collectAll() {
+    Callback<ResultDoubleObserver> currentMetricUpdater = metricUpdater;
+    if (currentMetricUpdater == null) {
+      return Collections.emptyList();
+    }
+    final ActiveBatcher activeBatcher = getActiveBatcher();
+    currentMetricUpdater.update(
+        new ResultDoubleObserver() {
```

Comment:
same comment here about having this be a concrete, rather than an anonymous class.

Reply:
Done.